job-service-acceptance-tests/pom.xml

```diff
@@ -254,13 +255,13 @@
                             <goal>start</goal>
                         </goals>
                     </execution>
-                    <execution>
+<!--                    <execution>
                         <id>stop</id>
                         <phase>post-integration-test</phase>
                         <goals>
                             <goal>stop</goal>
                         </goals>
-                    </execution>
+                    </execution>-->
```

Comment:
Why would you be commenting this out?  This makes me think this is not seriously ready for review.